AppointmentReminders.Web/Scripts/bootstrap-datetimepicker.js

```diff
@@ -0,0 +1,2426 @@
+/*! version : 4.14.30
```

Comment:
We should at the very least keep the minified version. You can ask Juan
Pedro and Eric what are the consequences of having dependencies in the
repository. They'll be happy to talk your ear off.

On Fri, Jul 17, 2015 at 11:10 AM, Agustin Camino <notifications@github.com>
wrote:

> In AppointmentReminders.Web/Scripts/bootstrap-datetimepicker.js
> <https://github.com/TwilioDevEd/appointment-reminders-csharp/pull/1#discussion_r34904313>
> :
>
> > @@ -0,0 +1,2426 @@
> > +/*! version : 4.14.30
>
> That could be an alternative, nonetheless I don't see a big impact in
> using a CDN or using a local copy of the library, I'll have a look in the
> extra dependencies in a while.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TwilioDevEd/appointment-reminders-csharp/pull/1/files#r34904313>
> .
>